pkgs/applications/office/softmaker/softmaker_office.nix

```diff
@@ -8,7 +8,7 @@ callPackage ./generic.nix (args // rec {
 
   src = fetchurl {
     url = "https://www.softmaker.net/down/softmaker-office-${edition}-${version}-amd64.tgz";
-    sha256 = "0wqaxng6s7lxwz6v2j6y1m5h4g4v63m0lscj7l2fpx5ksjlamp55";
+    sha256 = "0s8kzpc6w2cjkfqmhb2p1pkmkiq9vk9gnc7z2992kijaf9bwk3qz";
```

Comment:
This means that they released a new version, the version should also be bumped up to 1016.

(Upstream makes tarball of the previous version a redirect to the new version, see: https://forum.softmaker.com/viewtopic.php?f=320&t=19327 )